charts/newrelic-infrastructure/templates/controlplane/_agent-config_helper.tpl

```diff
@@ -0,0 +1,20 @@
+{{- /*
+Defaults for controlPlane's agent config
+*/ -}}
+{{- define "nriKubernetes.controlPlane.agentConfig.defaults" -}}
```

Comment:
I see, what i'm not convinced is about having the default inside the template, I don't know if there are good practices or standards related but I like more the idea to put defaults on Values or the templates rather than the helplers.

Reply:
I thought it was a good idea and have this in a more declarative way here in a helper instead of having it configured in three different layers (environment variables in Dockerfile, environment variables at container runtime and a couple of lines in the manifest)

The point of this is that these defaults are not "defaults" but are needed for the integration to work. They were hardcoded as environment variables and inside the forwarder image. That is the reason I removed them from being "hidden" between many layers and moved here.

The map for agent config is only in `.Values.common` so there is no possibility to make it go to the values unless we add config per scraper but I would be happy to do that change if that makes things easy to follow, document and reuse possible values.

Let me know what you think :)